lib/request.js

```diff
@@ -257,15 +257,14 @@ module.exports = {
    */
 
   get fresh() {
-    const method = this.method;
-    const s = this.ctx.status;
+    const { method, ctx: { status: s, response }, header } = this;
```

Comment:
Removed destructions renaming (it was to keep original code below), and it looks pretty neat now